components/telemetry-operator/apis/telemetry/v1alpha1/logpipeline_types.go

```diff
@@ -50,30 +51,38 @@ type ApplicationInput struct {
 	DropLabels bool `json:"dropLabels,omitempty"`
 }
 
+// InputNamespaces describes whether application logs from specific namespaces should be selected. The options are mutually exclusive. System namespaces are excluded by default from the collection
 type InputNamespaces struct {
+	// Include describes to include only the container logs of the specified namespace names
 	Include []string `json:"include,omitempty"`
+	// Exclude describes to exclude only the container logs of the specified namespace names
 	Exclude []string `json:"exclude,omitempty"`
-	System  bool     `json:"system,omitempty"`
+	// System describes to include the container logs of the system namespaces like kube-system, istio-system and kyma-system
+	System bool `json:"system,omitempty"`
 }
 
+// InputContainers describes whether application logs from specific containers should be selected. The options are mutually exclusive.
 type InputContainers struct {
+	// Include describes to include only the container logs with the specified container names
 	Include []string `json:"include,omitempty"`
+	// Exclude describes to exclude only the container logs with the specified container names
 	Exclude []string `json:"exclude,omitempty"`
 }
 
-// Filter describes a Fluent Bit filter configuration
+// Filter describes a filtering option on the logs of the pipeline.
```

Comment:
```suggestion
// Filter describes a filtering option on the logs of the pipeline
```

All other lines have no full stop, why here?